frontend/www/js/omegaup/components/qualitynomination/List.test.ts

```diff
@@ -0,0 +1,62 @@
+import { shallowMount } from '@vue/test-utils';
+import expect from 'expect';
+import Vue from 'vue';
+
+import T from '../../lang';
+import * as UI from '../../ui';
+
+import list from './List.vue';
+import { Contest } from '../../api';
+
+describe('List.vue', () => {
+  it('Should handle empty list of nominations', async () => {
+    const wrapper = shallowMount(list, {
+      propsData: {
+        data: {
+          nominations: {
```

Comment:
```suggestion
          nominations: <types.NominationListItem[]>[
              <types.NominationListItem>{
```

así el typechecker puede hacer algo con respecto a los tipos.

Reply:
listo